lighthouse-core/scripts/gcp-collection/README.md

```diff
@@ -0,0 +1,20 @@
+# Usage
+
+1. Spin up machines and start the collection.
+   1. (Prerequiste) Install GCloud SDK and authenticate locally, Run `brew cask install google-cloud-sdk && gcloud auth login`
+   1. (Optional) Run `export TARGET_GIT_REF=<a lighthouse git ref that has been pushed> ` if wanting to run on anything but master.
+   1. (Optional) Run `export TARGET_RUNS=<a number> ` if needing more than 1 run per URL.
+   1. (Optional) Run `export LIGHTHOUSE_COLLECTION_GCLOUD_PROJECT=<project name> ` to set your GCloud project.
+   1. Run `bash run.sh`
```

Comment:
do you have to be in this folder? It'd be nice to give the instruction assuming the user is in the project root.

Reply:
> It'd be nice to give the instruction assuming the user is in the project root.

Feels a bit weird to me assuming you'd be in this folder for the readme. Looking around it seems we're pretty inconsistent on whether we write the subreadme as local-relative or project root relative 🤷 ok though